modules/ossm-upgrade-known-issues.adoc

```diff
@@ -7,6 +7,8 @@
 
 Known issues that may affect your upgrade include:
 
+* When upgrading an Operator, custom configurations for Jaeger or Kiali may be reverted. Before upgrading an Operator, note any custom configuration settings for the Jaeger or Kiali objects in the {SMProductShortName} production deployment so that you can recreate them.
```

Comment:
Replace "may be" with "might be". 
Per ISG "To indicate possibility, use 'might'. To indicate permission, use 'may'."

Reply:
Thanks for reviewing @sabrinajess. I've updated the PR.